test/pthread/exit.c

```diff
@@ -0,0 +1,29 @@
+
+// @expect verified
+
+#include <pthread.h>
+
+int x = 1;
+
+void *t1(void *arg)
+{
```

Comment:
Sorry...  I didn't know there was one regarding curly braces.

This is fixed in next commit.